controllers/user.controller.js

```diff
@@ -22,6 +22,16 @@ exports.load = async (req, res, next, id) => {
   }
 };
 
+exports.getUsers = async (req, res, next) => {
+  try {
+    let users = await User.find({});
+    users = [...users].map(user => user.transform());
+    return res.json(sendResponse(httpStatus[200], 'Request for all users sucessful', users, null));
```

Comment:
I suggest you stick to a common practice, that way there is consistency
```suggestion
    return res.json(sendResponse(httpStatus.OK, 'Request for all users successful', users, null));
```

Reply:
this would pass but please change it on your next push